onboarding/source/controller.c

```diff
@@ -97,16 +116,27 @@ static void controllerTask(void * pvParameters) {
     obc_error_code_t lightServiceStatus = initLightService();
     if (lightServiceStatus != OBC_ERR_CODE_SUCCESS) {
         /* USER CODE BEGIN */
-        // Deal with error when initializing light service task and/or queue
+        sciPrintf("%s\n","Error: Light service was not properly initialized");
```

Comment:
Instead of this, you can just write `sciPrintf("Error: Light service was not properly initialized\n");`

Reply:
This applies to the other times you print error messages as well. Although, this isn't a big deal so don't worry about it for this challenge.